File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    let num=nums1.concat(nums2)
    num=num.sort((a,b) => a-b)
    if( num.length%2==1){
        return num[Math.floor(num.length/2)]
    } if(num.length%2==0){
        return (num[num.length/2]+num[num.length/2 -1])/2
    }
};

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        for i in range(len(nums2)):
            nums1.append(nums2[i])
        print(nums1)
        nums1.sort()
        print(nums1)
        if len(nums1) % 2 == 0:
            return (nums1[len(nums1)//2] + nums1[len(nums1)//2 - 1]) / 2
        else:
            return (nums1[len(nums1)//2])
